scripts/build.py

```diff
@@ -110,21 +110,38 @@
 #### Telegraf Functions
 ################
 
+def print_banner():
```

Comment:
The formatting looks off in the file due the escaping. This is what it looks like when being run:
```
 _____     _                       __
/__   \___| | ___  __ _ _ __ __ _ / _|
  / /\/ _ \ |/ _ \/ _` | '__/ _` | |_
 / / |  __/ |  __/ (_| | | | (_| |  _|
 \/   \___|_|\___|\__, |_|  \__,_|_|
                  |___/
 Build Script
```
I added the banner to make it easy to tell which product is being run, but we can swap it with something else if you want it to show something else.